tools/manifest/sourcefile.py

```diff
@@ -264,6 +264,8 @@ def name_is_non_test(self):
                 self.name_prefix("MANIFEST") or
                 self.filename == "META.yml" or
                 self.filename.startswith(".") or
+                self.filename == "OWNERS" or
+                self.filename == "META.yml" or
```

Comment:
This value is already included in the expression

Reply:
Hah, I never noticed that when I reviewed this before. I suck.